src/Components/Common/Sidebar/Sidebar.tsx

```diff
@@ -219,17 +224,26 @@ interface ToggleShrinkProps {
 }
 
 const ToggleShrink = ({ shrinked, toggle }: ToggleShrinkProps) => (
-  <div
-    className={`flex h-5 w-5 cursor-pointer items-center justify-center self-end rounded bg-gray-300 text-secondary-100 text-opacity-70 hover:bg-primary-700 hover:text-opacity-100 ${
-      shrinked ? "mx-auto" : "mr-4"
-    } transition-all duration-200 ease-in-out`}
-    onClick={toggle}
-  >
-    <CareIcon
-      icon="l-angle-up"
-      className={`text-3xl ${
-        shrinked ? "rotate-90" : "-rotate-90"
-      } transition-all delay-150 duration-300 ease-out`}
-    />
-  </div>
+  <TooltipProvider>
+    <Tooltip>
+      <TooltipTrigger asChild>
+        <button
+          className={`flex h-6 w-6 cursor-pointer items-center justify-center rounded focus:outline-none focus:ring-2 focus:ring-indigo-500 ${shrinked ? "bg-gray-200" : "bg-gray-100"} text-gray-600 hover:bg-primary-200 hover:text-primary-800 ${
+            shrinked ? "mx-auto" : "mr-4"
+          } transition-all duration-200 ease-in-out`}
+          onClick={toggle}
+        >
+          <CareIcon
+            icon={shrinked ? "l-arrow-bar-right" : "l-layout-sidebar-alt"}
+            className={`text-lg ${
+              shrinked ? "" : ""
+            } transition-all delay-150 duration-300 ease-out`}
```

Comment:
@vinutv  do we need this check? it's not applying any classes in both situations right. or was something else intended here?

Reply:
This check is unnecessary and was missed during cleanup. It has now been removed. [82091c2](https://github.com/ohcnetwork/care_fe/pull/8789/commits/82091c2e3f9d03dd587cf4d187c629d6ccc37161)